app/views/users/index.html.erb

```diff
@@ -1,8 +1,16 @@
-<h1>List of all static users:</h1>
-<ul>
-  <li>User 1</li>
-  <li>User 2</li>
-  <li>User 3</li>
-  <li>User 4</li>
-  <li>User 5</li>
-</ul>
+<section class="users">
+  <ul class="user-list">
+    <% @users.each do |user| %>
+      <%= link_to user_path(user) do %>
+      <li class="user-item">
+        <div class="image-box"><img src="<%= user.photo %>" alt="user photo"></div>
+        <div class="user-info-box">
+          <h1 class="user-name"><%= user.name %></h1>
+          <h4 class="posts-count">Number of posts: <%= user.posts.count %></h4>
+        </div>
+
+      </li>
+      <% end %>
+    <% end %>
+  </ul>
+</section>
```

Comment:
- [ ] Great job so far @Alibaba2023 There are just a few changes I will suggest you implement and you'll be on your way to the next MIlestone :wink: 
- [ ] As the previous reviewer mentioned, kindly ensure that you follow the given wireframe. From the video below, you can see that the image wasn't styled as seen in the wireframe. As a result when I created a user with the image, it affected the rest of the layout of each card which shouldn't be the case so kindly fix that :+1:
- [ ] KIndly make the same changes on the `user/show` page as well :+1:  

[Screencast from 11-03-2023 01:47:17 PM.webm](https://github.com/Alibaba2023/blog_app/assets/102699958/4630fbcd-c247-4fbd-8618-ac74e0a2dc65)

- THE GIVEN WIREFRAME
![image](https://github.com/Alibaba2023/blog_app/assets/102699958/7dedffc3-aca0-49e0-9953-4bc2b21e6716)